apps/govern/components/VeOlas/IncreaseLockModal/IncreaseUnlockTime.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Divider, Flex, Form } from 'antd';
+import { useState } from 'react';
+
+import { notifyError, notifySuccess } from 'libs/util-functions/src';
+
+import {
+  dateInMs,
+  getRemainingTimeInSeconds,
+  updateIncreaseUnlockTime,
+} from 'common-util/functions';
+import { useFetchBalances } from 'hooks/useFetchBalances';
+
+import { ProjectedVeOlas } from '../ProjectedVeOlas';
+import { UnlockTimeInput } from '../UnlockTimeInput';
+import { useVeolasComponents } from '../useVeolasComponents';
+
+type IncreaseUnlockTimeProps = {
+  closeModal: () => void;
+};
+
+type FormValues = {
+  unlockTime: number;
+};
+
+export const IncreaseUnlockTime = ({ closeModal }: IncreaseUnlockTimeProps) => {
+  const [form] = Form.useForm();
+  const { account, lockedEnd, olasBalance, veOlasBalance, refetch } = useFetchBalances();
+  const { getUnlockTimeComponent } = useVeolasComponents();
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const unlockTime = dateInMs(Form.useWatch('unlockTime', form));
+
+  const onFinish = async ({ unlockTime }: FormValues) => {
```

Comment:
same here, suggesting `handleFinish`, or perhaps a name more aligned with what the function does